dev/io.openliberty.netty.internal/bnd.bnd

```diff
@@ -12,27 +12,26 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
-Bundle-Name: Liberty Netty Transport Implementation
+Bundle-Name: Liberty Netty Transport Framework
 Bundle-SymbolicName: io.openliberty.netty.internal
-Bundle-Description: Liberty Netty Transport Implementation ${bVersion}
-WS-TraceGroup: ChannelFramework
+Bundle-Description: Liberty Netty Transport Framework ${bVersion}
+WS-TraceGroup: Netty
 
--dsannotations: io.openliberty.netty.internal.NettyFramework,\
+Export-Package: \
+  io.openliberty.netty.internal, \
```

Comment:
Having 'internal' in the exported packages is understood - we will just have to remember that using
the buildpath and auto-generated Import list in clients' bnds will skip these.